recipes/r-keys/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set version = '0.1.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-keys
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/keys_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/keys/keys_{{ version }}.tar.gz
+  sha256: 2b450b72579386e91ce6d1ca5b2d5bac55e116f952f8c8f2ecea760fc161074b
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-htmltools
+    - r-jsonlite
+    - r-shiny
+  run:
+    - r-base
+    - r-htmltools
+    - r-jsonlite
+    - r-shiny
+
+test:
+  commands:
+    - $R -e "library('keys')"           # [not win]
+    - "\"%R%\" -e \"library('keys')\""  # [win]
+
+about:
+  home: https://github.com/r4fun/keys
+  license: Apache-2.0
+  summary: Assign and listen to keyboard shortcuts in 'shiny' using the 'Mousetrap' Javascript
+    library.
+  license_family: APACHE
+  license_file: LICENSE.md
+
+extra:
+  recipe-maintainers:
+    - hechth
```

Please also add the conda-forge/r group, they help to maintain the conda-forge R packages. Thanks
```suggestion
    - conda-forge/r
    - hechth
```